examples/bfetch_explorer/public/components/page/index.tsx

```diff
@@ -44,7 +44,7 @@ export interface PageProps {
 
 export const Page: React.FC<PageProps> = ({ title = 'Untitled', children }) => {
   return (
-    <EuiPageBody>
+    <EuiPageBody component="main">
```

Comment:
The default used to `main` but is `div` now. All these `component="main"` additions are to make it use `main`.